diamm/helpers/solr_pagination.py

```diff
@@ -148,6 +144,16 @@ def type_list(self):
         return OrderedDict(filtered_facets)
 
     @property
+    def images_count(self):
+        image_count = self.result.facets['facet_fields']['public_images_b']
```

Comment:
This should probably be converted to use `get()` since it will raise a `KeyError` if (for some reason) it's not in the response.